.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: ✨ Feature request
+about: Template for requesting new features
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+## 📚 Context
+
+What is the problem that you are trying to solve?
+Why is this problem relevant?
+
+## ✔️ Solution
+
+(This section and the ones below are optional.)
+What are the possible solutions?
+If there are multiple alternatives, what are the benefits and drawbacks of each one?
+
+## 📈 Subtasks
+
+- [ ] If there is a solution, what are the subtasks for completing this issue?
+
+## 🎯 Definition of Done
+
+- [ ] If there is a solution, what are the final deliverables?
```

Comment:
@torives, after writing many issues, I thought his section had become useless. Now I want to remove it. What do you think?

Reply:
I like how it can work as a summary of the acceptance criteria for the issue. But for simpler issues, it can get repetitive pretty fast and most of it is already present on the Tasks and Solution sections anyway. I don't oppose its removal 😉